pages/common/redis-server.md

```diff
@@ -0,0 +1,16 @@
+# redis-server
+
+> Persistent key-value database.
+> More information: <https://redis.io/>.
+
+- Start redis daemon, in the foreground, with the default port (6379):
+
+`redis-server`
+
+- Start redis daemon, as a daemon (background process), with the default port:
+
+`redis-server --daemonize yes`
+
+- Start redis daemon, as a daemon, on a non-default port:
```

Comment:
```suggestion
- Start Redis server in a specific port as a background process:
```

Reply:
I made a change consistent with your suggestion, but I slightly changed the english around the port number.